build_docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
build_docker and start_docker can be in a Makefile in the future.

Reply:
Yes, that could be done.